x/fungible/keeper/foreign_coins.go

```diff
@@ -81,20 +81,18 @@ func (k Keeper) GetAllForeignCoins(ctx sdk.Context) (list []types.ForeignCoins)
 	return
 }
 
-// GetAllForeignERC20CoinMap returns all foreign ERC20 coins in a map of chainID -> asset -> coin
-func (k Keeper) GetAllForeignERC20CoinMap(ctx sdk.Context) map[int64]map[string]types.ForeignCoins {
+// GetAllForeignCoinMap returns all foreign ERC20 coins in a map of chainID -> asset -> coin
+func (k Keeper) GetAllForeignCoinMap(ctx sdk.Context) map[int64]map[string]types.ForeignCoins {
```

Comment:
Seems this function is not tested

Reply:
https://github.com/zeta-chain/node/pull/2060/commits/fa24f32191e070f8729095afe027e573ac1a3d51